index.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express')
+const { getAllMovies, getMovie, saveNewMovie } = require('./controllers/movies')
+const app = express()
+const bodyParser = require('body-parser')
```

Comment:
Should have all of your imports first and then a newline before starting your code.

```JavaScript
const express = require('express')
const bodyParser = require('body-parser')
const { getAllMovies, getMovie, saveNewMovie } = require('./controllers/movies')

const app = express()
```